docs/manual/src/internals/bindings_ir_pipeline.md

```diff
@@ -65,6 +65,33 @@ pub struct Node {
 }
 ```
 
+### Field Conversion order
+
+Fields are converted in declaration order.
+This matters when a field one uses `#[node(from(<name_of_the_another_field>)]`.
```

Comment:
This is much clearer, thanks!

Maybe "This matters when a field is listed twice with one of them using `#[node(from(<name_of_the_another_field>)]`"?

(I'm still not quite sure what the semantics would be of the `#[node(from(<name_of_the_another_field>)]` field was listed second and the non-option item was listed first - how does the option get converted to the non-option there?

Reply:
That wording sounds good, thanks.

> (I'm still not quite sure what the semantics would be of the #[node(from(<name_of_the_another_field>)] field was listed second and the non-option item was listed first - how does the option get converted to the non-option there?

That would be the same as the general case of converting an optional value into a non-option.  It will always fail -- even if there were no None values (I'm not 100% sure on that, but I'm pretty sure that's how it should work).